site/src/pages/docs/server-grpc.mdx

```diff
@@ -227,11 +227,45 @@ for JSON POST body.
 ## Blocking service implementation
 
 Unlike upstream gRPC-Java, Armeria does not run service logic in a separate thread pool by default. If your
-service implementation requires blocking, either run the individual blocking logic in a thread pool, wrap the
-entire service implementation in `RequestContext.current().blockingTaskExecutor().submit`, or set
+service implementation requires blocking, either run the individual blocking logic in a thread pool,
```

Comment:
It should be nice if we provide examples for each use case.
What do you think of listing all options with sample code?
```
### Using <type://@Blocking> annotation

... 

### Wrapping a service with <type://ServiceRequestContext#blockingTaskExecutor()>

...

### Using <type://GrpcServiceBuilder#useBlockingTaskExecutor(boolean)>

... // A builder example.
```

Note that `.blockingTaskExecutor()` only exists in `ServiceRequestContext`. The original documentation is incorrect. You may want to consider it. 😉